src/zenml/cli/base.py

```diff
@@ -433,3 +437,80 @@ def _prompt_email(event_source: AnalyticsEventSource) -> bool:
         )
 
     return False
+
+
+@cli.command(
+    "info", help="Show information about the current user setup.", hidden=True
+)
+@click.option(
+    "--packages",
+    "-p",
+    multiple=True,
+    help="Select specific installed packages.",
+    type=str,
+)
+@click.option(
+    "--all",
+    "-a",
+    is_flag=True,
+    default=False,
+    help="Output information about all installed packages.",
+    type=bool,
+)
+@click.option(
+    "--file",
+    "-f",
+    is_flag=True,
+    help="Output to a file.",
+    type=bool,
+)
+def info(packages, all, file) -> None:
+    """Show information about the current user setup.
+
+    Args:
+        packages: List of packages to show information about.
+        all: Flag to show information about all installed packages.
+        file: Flag to output to a file.
+    """
+    gc = GlobalConfiguration()
+    environment = Environment()
+    client = Client()
+    store_info = client.zen_store.get_store_info()
+
+    store_cfg = gc.store
+
+    user_info = {
+        "zenml_local_version": zenml_version,
+        "zenml_server_version": store_info.version,
+        "zenml_server_database": str(store_info.database_type),
+        "zenml_server_deployment_type": str(store_info.deployment_type),
+        "zenml_config_dir": gc.config_directory,
+        "zenml_local_store_dir": gc.local_stores_path,
+        "zenml_server_url": store_cfg.url,
+        "zenml_active_repository_root": str(client.root),
+        "python_version": environment.python_version(),
+        "environment": get_environment(),
+        "system_info": environment.get_system_info(),
+        "active_workspace": client.active_workspace.name,
+        "active_stack": client.active_stack_model.name,
+        "active_user": client.active_user.name,
+        "telemetry_status": "enabled" if gc.analytics_opt_in else "disabled",
+        "analytics_client_id": str(gc.user_id),
+        "analytics_user_id": str(client.active_user.id),
+        "analytics_server_id": str(client.zen_store.get_store_info().id),
+        "integrations": integration_registry.get_installed_integrations(),
+    }
+
+    if all:
+        user_info["packages"] = cli_utils.get_installed_local_packages()
+    if packages:
```

Comment:
Could this be an `elif`? If all packages are already included above, I probably don't need this extra subset here?

Reply:
There are so many packages. I can imagine wanting to have them all, but also wanting to have a special place just to see the subset I care about right now. At least that was what I was going for here.